src/models/__mocks__/article.js

```diff
@@ -0,0 +1,54 @@
+export const normalArticle = {
```

Comment:
i added naming convention for mocks on our readme, we will now call mocks like mock*** .. for instance change this to mockArticle and change all the other stuff

Reply:
changed